### ISSUE TITLE
Cannot Post Bug to JIRA

### ISSUE THREAD

**Issue by byip-ujet** [reportportal/reportportal#865]
**Describe the bug**
On the test failure screen, we attempt to Post Issue but receive an error
`Error occurred: {"error_code":40302,"message":"Impossible interact with external system. org.codehaus.jettison.json.JSONException: A JSONObject text must begin with '' at character 1 of <html>\n\n<head>\n \n\n\n\n\n\n\n <title>Unauthorized (401)</title>\n \n\n\n\n\n\n<!--[if IE]><![endif]-->\n<script type=\"text/javascript\" >\n var contextPath = '';\n var DeferScripts = deferState: 'disabled' ;\n</script>\n<script>\nwindow.WRM=window.WRM||;window.WRM._unparsedData=window.WRM._unparsedData||;window.WRM._unparsedErrors=window.WRM._unparsedErrors||;\nWRM._unparsedData[\"com.atlassian.plugins.atlassian-plugins-webresource-plugin:context-path.context-path\"]=\"\\\"\\\"\";\nWRM._unparsedData[\"com.atlassian.plugins.atlassian-plugins-webresource-rest:curl.cross-origin-resources\"]=\"false\";\nWRM._unparsedData[\"jira.webresources:feature-flags.feature-flag-data\"]=\"\\\"enabled-feature-keys\\\":[\\\"jira.plugin.devstatus.smartcommits.dvcs.enabled\\\",\\\"nps.survey.inline.dialog\\\",\\\"jira.export.excel.enabled\\\",\\\"com.atlassian.jira.plugins.mail.internal.enabled\\\",\\\"sd.new.settings.sidebar.location\\\",\\\"re-enable.group.endpoints.jra-60287\\\",\\\"dvcs.connector.smartcommits.disabled\\\",\\\"unified.usermanagement\\\",\\\"com.atlassian.jira.projects.issuenavigator\\\",\\\"jira.issue.search.jql.indexprimary.enabled\\\",\\\"jira.issue.search.api.indexprimary.enabled\\\",\\\"com.atlassian.jira.projects.issue.summary.vignettes\\\",\\\"atlassian.aui.raphael.disabled\\\",\\\"fusion.dvcs.use_amq_for_executors.enabled\\\",\\\"com.atlassian.jira.config.CoreFeatures.PERMISSIONS_MANAGED_BY_UM\\\",\\\"com.atlassian.jira.projects.ProjectCentricNavigation.Switch\\\",\\\"com.atlassian.jira.config.CoreFeatures.ON_DEMAND\\\",\\\"atlassian.plugin.aci.enabled\\\",\\\"connect.no-applinks\\\",\\\"fusion.dvcs.use_amq_to_invoke_sync.enabled\\\",\\\"com.atlassian.portfolio.classicPlans.sunset\\\",\\\"com.atlassian.rm.portfolio.vertigo.issueloading.Database\\\",\\\"jira.baseurl.cdn.enabled\\\",\\\"fusion.bitbucket.rebrand\\\",\\\"com.atlassian.portfolio.integrations.ClassicPlans.disabled\\\",\\\"connect.authentication.oauth2.impersonation\\\",\\\"sd.email.mailbox.poppy.migration.poppyonly.enabled\\\",\\\"dvcs.connector.repositorydao.ao.enabled\\\",\\\"fusion.devstatus.devsummary.customfield.jql.enabled\\\"],\\\"feature-flag-states\\\":\\\"jira.issue.search.jql.anonymize\\\":true,\\\"jira.feature.flag.board.issue.modal.labs\\\":true,\\\"com.atlassian.jira.plugins.mail.internal\\\":false,\\\"com.atlassian.jira.feature.flag.for.TestFeatureFlagResource\\\":false,\\\"jira.mail.default.mailstore.polling\\\":true,\\\"com.atlassian.jira.plugins.mail.size.check\\\":true,\\\"jira.instrumentation.laas\\\":false,\\\"jira.feature.rest.single.step.avatar.upload\\\":true,\\\"jira.feature.flag.board.polling\\\":true,\\\"issue.collector.cache.limiter\\\":true,\\\"jira.login.useBasicAuthFilter\\\":true,\\\"jira.ondemand.maintain.trusted.apps\\\":true,\\\"jira.mail.fake.default.mail.server\\\":false\";\nWRM._unparsedData[\"jira.webresources:default-comment-security-level.DefaultCommentSecurityLevelHelpLink\"]=\"\\\"extraClasses\\\":\\\"default-comment-level-help\\\",\\\"title\\\":\\\"Commenting on an Issue\\\",\\\"url\\\":\\\"https://confluence.atlassian.com/display/JIRACORECLOUD/Editing+and+collaborating+on+issues#Editingandcollaboratingonissues-addcommentsCommentingonissues\\\",\\\"isLocal\\\":false\";\nWRM._unparsedData[\"com.atlassian.jira.jira-client-analytics-plugin:analytics-context-provider.client-analytic-descriptors\"]=\"\\\"analyticsResource\\\":\\\"jira-core\\\":\\\"events\\\":\\\"issue.comment.added\\\":\\\"trigger\\\":\\\"click #issue-comment-add-submit\\\",\\\"properties\\\":\\\"comment-length\\\":\\\"value.length #issue-comment-add #comment\\\",\\\"tenantId\\\":\\\"2e1b2946-b205-4a5f-aaca-b5ab048f988c\\\",\\\"env\\\":\\\"prod\\\"\";\nWRM._unparsedData[\"jira.webresources:key-commands.shortcuts\"]=\"[\\\"moduleKey\\\":\\\"go-to-dashboard-kb-shortcut\\\",\\\"keys\\\":[[\\\"g\\\",\\\"d\\\"]],\\\"context\\\":\\\"global\\\",\\\"op\\\":\\\"goTo\\\",\\\"param\\\":\\\"/secure/Dashboard.jspa\\\",\\\"moduleKey\\\":\\\"browse-project-kb-shortcut\\\",\\\"keys\\\":[[\\\"g\\\",\\\"p\\\"]],\\\"context\\\":\\\"global\\\",\\\"op\\\":\\\"execute\\\",\\\"param\\\":\\\"var elem = jQuery(\\\\\\\"#browse_link\\\\\\\");\\\\n if (elem.length \\u003e 0) \\\\n window.location.href = elem.attr(\\\\\\\"href\\\\\\\");\\\\n \\\",\\\"moduleKey\\\":\\\"issue-navigator-kb-shortcut\\\",\\\"keys\\\":[[\\\"g\\\",\\\"i\\\"]],\\\"context\\\":\\\"global\\\",\\\"op\\\":\\\"execute\\\",\\\"param\\\":\\\"var elem = jQuery(\\\\\\\"#find_link\\\\\\\");\\\\n if (elem.length \\u003e 0) \\\\n window.location.href = elem.attr(\\\\\\\"href\\\\\\\");\\\\n \\\",\\\"moduleKey\\\":\\\"admin.quicksearch.dialog.shortcut\\\",\\\"keys\\\":[[\\\"g\\\",\\\"g\\\"]],\\\"context\\\":\\\"global\\\",\\\"op\\\":\\\"execute\\\",\\\"param\\\":\\\"JIRA.Shifter.show();\\\",\\\"moduleKey\\\":\\\"quicksearch\\\",\\\"keys\\\":[[\\\"/\\\"]],\\\"context\\\":\\\"global\\\",\\\"op\\\":\\\"moveToAndFocus\\\",\\\"param\\\":\\\"#quickSearchInput, #admin-quick-nav-field\\\",\\\"moduleKey\\\":\\\"create.issue\\\",\\\"keys\\\":[[\\\"c\\\"]],\\\"context\\\":\\\"global\\\",\\\"op\\\":\\\"moveToAndClick\\\",\\\"param\\\":\\\"#header .create-issue\\\",\\\"moduleKey\\\":\\\"shifter\\\",\\\"keys\\\":[[\\\".\\\"]],\\\"context\\\":\\\"global\\\",\\\"op\\\":\\\"execute\\\",\\\"param\\\":\\\"JIRA.Shifter.show();\\\",\\\"moduleKey\\\":\\\"help\\\",\\\"keys\\\":[[\\\"?\\\"]],\\\"context\\\":\\\"global\\\",\\\"op\\\":\\\"click\\\",\\\"param\\\":\\\"#keyshortscuthelp\\\"]\";\nWRM._unparsedData[\"com.atlassian.plugins.atlassian-connect-plugin:dialog-options.data\"]=\"\\\"allJsVersion\\\":\\\"1.2.29\\\",\\\"dialogOptions\\\":\\\"com.atlassian.jira.spreadsheets__open-in-excel\\\":\\\"height\\\":\\\"365px\\\",\\\"chrome\\\":false,\\\"width\\\":\\\"648px\\\",\\\"com.atlassian.jira.spreadsheets__open-in-gsheets\\\":\\\"height\\\":\\\"365px\\\",\\\"chrome\\\":false,\\\"width\\\":\\\"648px\\\",\\\"inlineDialogOptions\\\":,\\\"dialogModules\\\":\\\"com.invisionapp.integration.jira\\\":\\\"expand\\\":\\\"cacheable\\\":false,\\\"options\\\":,\\\"key\\\":\\\"expand\\\",\\\"auth\\\":\\\"cacheable\\\":false,\\\"options\\\":,\\\"key\\\":\\\"auth\\\",\\\"stspg-jira-ops\\\":\\\"login-dialog\\\":\\\"cacheable\\\":false,\\\"options\\\":\\\"height\\\":\\\"700px\\\",\\\"chrome\\\":false,\\\"header\\\":\\\"value\\\":\\\"Statuspage Login\\\",\\\"width\\\":\\\"600px\\\",\\\"key\\\":\\\"login-dialog\\\",\\\"create-incident-dialog\\\":\\\"cacheable\\\":false,\\\"options\\\":\\\"height\\\":\\\"100%\\\",\\\"chrome\\\":false,\\\"header\\\":\\\"value\\\":\\\"Create Incident\\\",\\\"width\\\":\\\"100%\\\",\\\"key\\\":\\\"create-incident-dialog\\\",\\\"update-incident-dialog\\\":\\\"cacheable\\\":false,\\\"options\\\":\\\"height\\\":\\\"100%\\\",\\\"chrome\\\":false,\\\"header\\\":\\\"value\\\":\\\"Update Incident\\\",\\\"width\\\":\\\"100%\\\",\\\"key\\\":\\\"update-incident-dialog\\\",\\\"project-configure-dialog\\\":\\\"cacheable\\\":false,\\\"options\\\":\\\"height\\\":\\\"700px\\\",\\\"chrome\\\":false,\\\"header\\\":\\\"value\\\":\\\"Configure\\\",\\\"width\\\":\\\"600px\\\",\\\"key\\\":\\\"project-configure-dialog\\\",\\\"ops-setup-flow\\\":\\\"cacheable\\\":false,\\\"options\\\":\\\"height\\\":\\\"700px\\\",\\\"chrome\\\":false,\\\"header\\\":\\\"value\\\":\\\"Configure\\\",\\\"width\\\":\\\"600px\\\",\\\"key\\\":\\\"ops-setup-flow\\\",\\\"change-project-statuspage\\\":\\\"cacheable\\\":false,\\\"options\\\":\\\"height\\\":\\\"700px\\\",\\\"chrome\\\":false,\\\"header\\\":\\\"value\\\":\\\"Change Project Statuspage\\\",\\\"width\\\":\\\"600px\\\",\\\"key\\\":\\\"change-project-statuspage\\\",\\\"com.atlassian.jira.emcee\\\":\\\"in-product-install-dialog\\\":\\\"cacheable\\\":false,\\\"options\\\":\\\"size\\\":\\\"large\\\",\\\"chrome\\\":false,\\\"key\\\":\\\"in-product-install-dialog\\\"\";\nWRM._unparsedData[\"com.atlassian.plugins.atlassian-connect-plugin:jira-date-picker-widget.config\"]=\"\\\"dateFormat\\\":\\\"%e/%b/%y\\\",\\\"dateTimeFormat\\\":\\\"%e/%b/%y %I:%M %p\\\",\\\"timeFormat\\\":\\\"12\\\",\\\"firstDay\\\":1,\\\"useISO8601WeekNumbers\\\":false\";\nWRM._unparsedData[\"com.atlassian.product.fabric.pf-jira-confluence-integration-plugin:app-switcher-initial-resources.cloud-id\"]=\"\\\"2e1b2946-b205-4a5f-aaca-b5ab048f988c\\\"\";\nWRM._unparsedData[\"com.atlassian.product.fabric.pf-jira-confluence-integration-plugin:app-switcher-initial-resources.is-anonymous\"]=\"true\";\nWRM._unparsedData[\"jira.webresources:dateFormatProvider.dateFormat\"]=\"\\\"meridiem\\\":[\\\"AM\\\",\\\"PM\\\"],\\\"eras\\\":[\\\"BC\\\",\\\"AD\\\"],\\\"months\\\":[\\\"January\\\",\\\"February\\\",\\\"March\\\",\\\"April\\\",\\\"May\\\",\\\"June\\\",\\\"July\\\",\\\"August\\\",\\\"September\\\",\\\"October\\\",\\\"November\\\",\\\"December\\\"],\\\"monthsShort\\\":[\\\"Jan\\\",\\\"Feb\\\",\\\"Mar\\\",\\\"Apr\\\",\\\"May\\\",\\\"Jun\\\",\\\"Jul\\\",\\\"Aug\\\",\\\"Sep\\\",\\\"Oct\\\",\\\"Nov\\\",\\\"Dec\\\"],\\\"weekdaysShort\\\":[\\\"Sun\\\",\\\"Mon\\\",\\\"Tue\\\",\\\"Wed\\\",\\\"Thu\\\",\\\"Fri\\\",\\\"Sat\\\"],\\\"weekdays\\\":[\\\"Sunday\\\",\\\"Monday\\\",\\\"Tuesday\\\",\\\"Wednesday\\\",\\\"Thursday\\\",\\\"Friday\\\",\\\"Saturday\\\"]\";\nWRM._unparsedData[\"com.atlassian.servicedesk.frontend-webpack-plugin:data-provider.webpack-meta\"]=\"\\\"locale\\\":\\\"en-US\\\",\\\"publicPath\\\":\\\"//d23nms05qbuy2b.cloudfront.net/atl-vertigo--shard-jira-prod-us-17--4--jres.atlassian.net/s/cfqidq/b/14/7d9baab32824ee172d27913984449c308c10d935/_/download/resources/com.atlassian.servicedesk.frontend-webpack-plugin:webpack-async-chunks/\\\"\";\nWRM._unparsedData[\"com.atlassian.applinks.applinks-plugin:applinks-common-exported.applinks-help-paths\"]=\"\\\"entries\\\":\\\"applinks.docs.root\\\":\\\"https://confluence.atlassian.com/display/APPLINKS-080/\\\",\\\"applinks.docs.diagnostics.troubleshoot.sslunmatched\\\":\\\"SSL+and+application+link+troubleshooting+guide\\\",\\\"applinks.docs.diagnostics.troubleshoot.oauthsignatureinvalid\\\":\\\"OAuth+troubleshooting+guide\\\",\\\"applinks.docs.diagnostics.troubleshoot.oauthtimestamprefused\\\":\\\"OAuth+troubleshooting+guide\\\",\\\"applinks.docs.delete.entity.link\\\":\\\"Create+links+between+projects\\\",\\\"applinks.docs.adding.application.link\\\":\\\"Link+Atlassian+applications+to+work+together\\\",\\\"applinks.docs.administration.guide\\\":\\\"Application+Links+Documentation\\\",\\\"applinks.docs.oauth.security\\\":\\\"OAuth+security+for+application+links\\\",\\\"applinks.docs.troubleshoot.application.links\\\":\\\"Troubleshoot+application+links\\\",\\\"applinks.docs.diagnostics.troubleshoot.unknownerror\\\":\\\"Network+and+connectivity+troubleshooting+guide\\\",\\\"applinks.docs.configuring.auth.trusted.apps\\\":\\\"Configuring+Trusted+Applications+authentication+for+an+application+link\\\",\\\"applinks.docs.diagnostics.troubleshoot.authlevelunsupported\\\":\\\"OAuth+troubleshooting+guide\\\",\\\"applinks.docs.diagnostics.troubleshoot.ssluntrusted\\\":\\\"SSL+and+application+link+troubleshooting+guide\\\",\\\"applinks.docs.diagnostics.troubleshoot.unknownhost\\\":\\\"Network+and+connectivity+troubleshooting+guide\\\",\\\"applinks.docs.delete.application.link\\\":\\\"Link+Atlassian+applications+to+work+together\\\",\\\"applinks.docs.link.applications\\\":\\\"Link+Atlassian+applications+to+work"}`

**Expected behavior**
It posts a ticket to our JIRA board.  This was working previously.

**Screenshots**
![Screen Shot 2019-08-27 at 11 04 24 AM](https://user-images.githubusercontent.com/41927305/63797426-ebfacd80-c8bc-11e9-8251-e9cd54bcfaaa.png)
![Screen Shot 2019-08-27 at 11 14 05 AM](https://user-images.githubusercontent.com/41927305/63797427-ebfacd80-c8bc-11e9-82e5-992d49102389.png)

**Versions:**
RP Build: 4.3.6
Mojave 10.14.6
Chrome 76.0.3809.132 (Official Build) (64-bit) AND 76.0.3809.100 (Official Build) (64-bit)

**Comment by evjlobanova** [comment]
@byip-ujet @sandy170889 is this still actual?
I am closing issue, feel free to raise it if you have problems with bts

**Comment by iivanou** [comment]
Transferring the issue back to the reportportal project. It is not related to the Python agent.

**Comment by jsingh-qualitrol** [comment]
@evjlobanova I was able to post issue to Jira successfully once. Instead of bug, Epic was configured in Jira plugin 5.0.1,so i had to change it back to bug. Now i am unable to post issue to JIra. I had verified the credential by login to jira. And trying to connect with Superadmin rights. I had reinstall jira plugin again to confirm but no luck. Can anyone help me with the issue. (edited) 
image.png
